Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.yml

```diff
@@ -0,0 +1,2860 @@
+      - problem investigation
+      - known error
+      - All
+  - name: ticket_status
+    display: Ticket Status
+    required: false
+    defaultvalue: All
+    type: 16
+    additionalinfo: The status of the tickets to fetch. Since Each ticket type has it's own unique status pool, consider to select only those who match to the selected ticket types.
+    options:
+      - Draft
+      - In Cart
+      - In Review
+      - Submitted
+      - Pending
+      - Waiting Approval
+      - Planning
+      - In Progress
+      - Completed
+      - Rejected
+      - Cancelled
+      - Closed
+      - New
+      - Assigned
+      - Resolved
+      - Request For Authorization
+      - Request For Change
+      - Planning In Progress
+      - Scheduled For Review
+      - Scheduled For Approval
+      - Scheduled
+      - Implementation In Progress
+      - Staged
+      - Work In Progress
+      - Waiting
+      - Bypassed
+      - Under Review
+      - Under Investigation
+      - Scheduled For Correction
+      - Assigned To Vendor
+      - No Action Planned
+      - Corrected
+      - All
+  - name: ticket_impact
+    display: Ticket Impact
+    required: false
+    defaultvalue: All
+    type: 16
+    additionalinfo: The impact of the tickets to fetch.
+    options:
+      - 1-Extensive/Widespread
+      - 2-Significant/Large
+      - 3-Moderate/Limited
+      - 4-Minor/Localized
+      - All
+  - name: ticket_urgency
+    display: Ticket Urgency
+    required: false
+    defaultvalue: All
+    type: 16
+    additionalinfo: The urgnecy of the tickets to fetch.
+    options:
+      - 1-Critical
+      - 2-High
+      - 3-Medium
+      - 4-Low
+      - All
+  - name: query
+    display: Fetch By Query
+    required: false
+    defaultvalue:
+    type: 0
+    additionalinfo: 'Search qualification to fetch tickets. For example: Status = "Draft" AND Impact = "1-Extensive/Widespread". The query will be used as an addition to the existing params. You can Review the BMC documentation for how to Build search qualifications: https://docs.bmc.com/docs/ars2008/building-qualifications-and-expressions-929630007.html'
+  - display: Fetch incidents
+    name: isFetch
+    required: false
+    type: 8
+  - additionalinfo: "Choose the direction to mirror the incident: Incoming (from  BMC Helix ITSM to XSOAR), Outgoing (from XSOAR to BMC Helix ITSM), or Incoming and Outgoing (from/to XSOAR and BMC Helix ITSM)."
+    defaultvalue: None
+    display: Incident Mirroring Direction
+    name: mirror_direction
+    options:
+      - None
+      - Incoming
+      - Outgoing
+      - Incoming And Outgoing
+    required: false
+    type: 15
+  - additionalinfo: When selected, closing the BMC Helix ITSM ticket is mirrored in Cortex XSOAR.
+    defaultvalue: "false"
+    display: Close Mirrored XSOAR Incident
+    name: close_incident
+    required: false
+    type: 8
+  - additionalinfo: When selected, closing the XSOAR incident is mirrored in BMC Helix ITSM.
```

Comment:
Need to add Cortex to the term XSOAR:
When selected, closing the Cortex XSOAR incident is mirrored in BMC Helix ITSM.